subsys/bluetooth/controller/ll_sw/ull_sync_iso.c

```diff
@@ -284,6 +291,19 @@ uint8_t ll_big_sync_terminate(uint8_t big_handle, void **rx)
 		return BT_HCI_ERR_CMD_DISALLOWED;
 	}
 
+	/* Do a blocking mayfly call to LLL context for flushing any outstanding
+	 * operations.
+	 */
+	flush_sem = &sem;
```

Comment:
```suggestion
	sync_iso->flush_sem = &sem;
```

Use the context to store instead of global variable to avoid race between two instances (possibly between thread requested terminate and timeout related sync lost?)

Reply:
Got that same commit into two different PRs. Removed from here, and concern addressed in #75952